src/openrct2/ride/Vehicle.cpp

```diff
@@ -10014,3 +9913,10 @@ const rct_vehicle* rct_vehicle::GetCar(size_t carIndex) const
     }
     return car;
 }
+
+void rct_vehicle::SetState(VEHICLE_STATUS vehicleStatus, uint8_t subState)
```

Comment:
I think a default value of 0 for the sub state would simplify this. The vast majority of situations it's 0

Reply:
I'll fix that.